code/modules/cargo/blackmarket/blackmarket_items/cybernetics.dm

```diff
@@ -0,0 +1,268 @@
+	item = /obj/item/organ/cyberimp/cyberlink/nt_high
+	availability_prob = 40
+	stock_min = 1
+	stock_max = 2
+	price_min = 300
+	price_max = 600
+
+/datum/blackmarket_item/cyberlinks/terragov
+	name = "Terragov Integrated Cybernetic Managment System"
+	desc = "Quite rare around this part of the galaxy, allows unrestricted connection to all of Terragov's implants, we don't guarantee it will work with any other cybernetic tough."
+	item = /obj/item/organ/cyberimp/cyberlink/terragov
+	availability_prob = 15
+	price_min = 700
+	price_max = 1000
+
+/datum/blackmarket_item/cyberlinks/syndicate
+	name = "Interdyne Cybernetic Operating Bioware"
+	desc = "A very rare cyberlink, used only by interdyne politicians, and other syndicate assault teams. Very rare find."
+	item = /obj/item/organ/cyberimp/cyberlink/syndicate
+	availability_prob = 5
+	price_min = 1000
+	price_max = 1200
+
+/// CYBERNETICS
+
+/datum/blackmarket_item/cybernetics
+	category = "Cybernetics"
+	markets = list(/datum/blackmarket_market/cybernetics)
+	stock = 1 //Can never have more than 1 of the same implant on the auction at the same time.
+	var/randomizable = TRUE
+
+/datum/blackmarket_item/cybernetics/spawn_item(loc)
+	. = ..()
+	var/obj/item/organ/cyberimp/implant = .
+	if(prob(30) && randomizable)
+		implant.random_encode()
+
+// LEGS
+
+/datum/blackmarket_item/cybernetics/table_glider
+	name = "Table-gliding cybernetic"
+	desc = "Works like a charm, can get you up any table in a second, don't ask if you can also quickly get down, we are still figuring that part out."
+	item = /obj/item/organ/cyberimp/leg/table_glider
+	availability_prob = 20
+	stock = 2 // ok, ok these require 2 implants on each leg to work
+	price_min = 200
+	price_max = 400
+
+/datum/blackmarket_item/cybernetics/shove_resist
+	name = "BU-TAM resistor cybernetic"
+	desc = "This bad boy roots you in ground whenever you get shoved, doesn't come with muscle actuators so your legs will get tired after a while."
+	item = /obj/item/organ/cyberimp/leg/shove_resist
+	availability_prob = 15
+	stock = 2 // ok, ok these require 2 implants on each leg to work
+	price_min = 300
+	price_max = 600
+
+/datum/blackmarket_item/cybernetics/accelerator
+	name = "P.R.Y.Z.H.O.K accelerator cybernetic"
+	desc = "Advanced russian implant that allows you to tackle people without special gloves or training, very rare to aquire these as TerraGov has cracked down on the sales."
+	item = /obj/item/organ/cyberimp/leg/accelerator
+	availability_prob = 10
+	stock = 2 // ok, ok these require 2 implants on each leg to work
+	price_min = 400
+	price_max = 800
+
+/datum/blackmarket_item/cybernetics/chemplant_drug
+	name = "deep-vein emergency morale rejuvenator"
+	desc = "Interdyne Pharmaceutic's cybernetic that was recenetly 'legally' aquired by us, now we are willing to sell a one of these bad boys to anyone i need of some rejuvenation!"
```

Comment:
shouldn't this say `anyone in need of some rejuvenation` instead of `anyone i need of some rejuvenation`